notebooks/nvidia_haystack_medical_chatbot.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<p>Why do we install both <code>nvidia-haystack</code> and another integration? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Writing a custom component didn't work when I only had `nvidia-haystack` installed. Maybe that's a bug?